src/lab10.py

```diff
@@ -10,19 +10,19 @@
 import tkinter.font
 import urllib.parse
 import dukpy
+import wbetools
 from lab2 import WIDTH, HEIGHT, HSTEP, VSTEP, SCROLL_STEP
 from lab3 import FONTS, get_font
 from lab4 import print_tree, HTMLParser
 from lab5 import BLOCK_ELEMENTS
 from lab6 import CSSParser, TagSelector, DescendantSelector
 from lab6 import INHERITED_PROPERTIES, style, cascade_priority
 from lab6 import DrawText, tree_to_list
-from lab7 import DrawLine, DrawOutline, DrawRect
-from lab8 import URL, Browser, Text, Element
+from lab7 import DrawLine, DrawOutline, DrawRect, Rect
+from lab8 import URL, Browser, Text, Element, Chrome
 from lab8 import BlockLayout, InputLayout, DEFAULT_STYLE_SHEET, INPUT_WIDTH_PX
-from lab8 import DocumentLayout, LineLayout, TextLayout
-from lab9 import EVENT_DISPATCH_JS, JSContext, Tab
-import wbetools
```

Comment:
Why is this no longer needed?

Reply:
Still needed, just moved up due to how the inliner works. (This is maybe a not-ideal way for the inliner to work but in this PR I just want the outline to work better.